subprojects/docs/build.gradle

```diff
@@ -775,18 +774,6 @@ tasks.named("docsTest") { task ->
                 "snippet-tasks-incremental-build-custom-task-class_kotlin_incrementalAdHocTaskNoSource.sample",
                 "snippet-tasks-incremental-build-custom-task-class_kotlin_inferredTaskDep.sample",
```

Comment:
:+1: 

Now that the snippets have been fixed, we can remove them from the list of snippets that are known to fail under configuration cache.